timemachine/ff/handlers/nonbonded.py

```diff
@@ -363,6 +375,18 @@ def static_parameterize(mol):
 
 
 class AM1BCCHandler(SerializableMixIn):
+    """The AM1BCCHandler generates charges for molecules using OpenEye's AM1BCCELF10[1] protocol.
+
+    Charges are conformer and platform dependent as of OpenEye Toolkits 2020.2.0 [2].
+
+    References
+    ----------
+    [1] AM1BCCELF10 Theory
+        https://docs.eyesopen.com/toolkits/python/quacpactk/molchargetheory.html#elf-conformer-selection
```

Comment:
Could we elaborate slightly on nature of conformer dependence for AM1BCC ELF10?

(I was under the impression the dependence on initial conformer was greatly reduced, or eliminated, by using ELF10.)

Reply:
Reading more about ELF10, I am not sure we really want this as it expects an ensemble of conformers. Which in most cases we don't provide. The way I interpret the documentation (https://docs.eyesopen.com/toolkits/python/quacpactk/molchargetheory.html#elf-conformer-selection) and this issue (https://github.com/openforcefield/openff-toolkit/issues/1170) is that when you have an ensemble of conformers ELF10 will provide a more 'accurate' set of partial charges. 

In general we only provide a single conformer and even if we provide a few I don't know that we "start with enough conformers so that it can find a population of conformers that do not have strongly-interacting functional groups". While it is not explicit in the docs, I assume it is falling back to AM1BCC when there is only a single conformer though I should check the behavior. 

> I was under the impression the dependence on initial conformer was greatly reduced, or eliminated, by using ELF10.

Will play around with this. I might just need to clarify that this statement means that the charges are not bitwise deterministic across conformer/platform, as it does seem that this would improve the charges accuracy, but use only 2% of the actual conformers.

Reply:
Thanks for clarifying this. I had misunderstood. (For some reason thought it was automatically generating more than 10 conformers to select from, if only 1 was provided initially...)

>While it is not explicit in the docs, I assume it is falling back to AM1BCC when there is only a single conformer though I should check the behavior.

This does appear to be the case! `oe_assign_charges(mol, 'AM1BCCELF10') == oe_assign_charges(mol, 'AM1BCC')` when `mol.GetNumConformers() == 1`, on some examples...

<details>
Script that might be helpful for playing around with examples:

```python
import numpy as np

from timemachine.ff.handlers.nonbonded import oe_assign_charges
from timemachine.datasets import fetch_freesolv

from rdkit import Chem
from rdkit.Chem import AllChem

def load_freesolv_mol_with_multiple_conformers(i, n_conformers=1, keep_initial_conformer=True):
    mol = fetch_freesolv()[i]
    
    assert mol.GetNumConformers() == 1
    
    initial_conformer = mol.GetConformer()
    AllChem.EmbedMultipleConfs(mol, n_conformers, clearConfs=not keep_initial_conformer)
    
    return mol

mol_i = 0

# single conformer case, dataset conformer vs. rdkit-generated conformer
mols = dict()
mols['a'] = load_freesolv_mol_with_multiple_conformers(mol_i, 1, False)
mols['b'] = load_freesolv_mol_with_multiple_conformers(mol_i, 0, True)

keys = ['a', 'b']
methods = ['AM1BCC', 'AM1BCCELF10']

for key in keys:
    assert mols[key].GetNumConformers() == 1

charges = dict()
for key in keys:
    for method in methods:
        charges[(key, method)] = oe_assign_charges(mols[key], method)
        
# AM1BCCELF10 == AM1BCC in single-conformer case
for key in keys:
    print(charges[(key, 'AM1BCC')] - charges[(key, 'AM1BCCELF10')])
    
# the two mols -- which differ only in conformer -- get assigned different charges
for method in methods:
    print(charges[('a', method)] - charges[('b', method)])

```
</details>

Will leave this "unresolved" for any further discussion, but this fully addresses my question.